bullet-list/README.md

```diff
@@ -0,0 +1,36 @@
+# Bullet list extractor examples
```

Comment:
better call the folder
extract-bullet-lists or something like that so it is more clear from high level what it does